Tests/CatalogItems/Rest/Get-RsRestItem.Tests.ps1

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2020 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+$reportPortalUri = if ($env:PesterPortalUrl -eq $null) { 'http://localhost/reports' } else { $env:PesterPortalUrl }
+$reportServerUri = if ($env:PesterServerUrl -eq $null) { 'http://localhost/reportserver' } else { $env:PesterServerUrl }
+
+Describe "Get-RsRestItem" {
+    Context "Get folder with reportPortalUri parameter"{
+        # Create a folder
+        $folderName = 'SutGetFolderReportPortalUriParameter' + [guid]::NewGuid()
+        New-RsRestFolder -reportPortalUri $reportPortalUri -RsFolder / -FolderName $folderName
+        $folderPath = '/' + $folderName
+        # Test if the folder can be found
+        $folderList = Get-RsRestItem -reportPortalUri $reportPortalUri -RsItem $folderPath
+        $folderCount = $folderList | Where-Object name -eq $folderName | measure
+        It "Should found a folder" {
+            $folderCount.Count | Should Be 1
+        }
+        # Removing folders used for testing
+        Remove-RsRestCatalogItem -ReportPortalUri $reportPortalUri -RsItem $folderPath -Confirm:$false
+    }
+Write-Host '----------------------------
```

Comment:
I would remove this or put them inside the {

Reply:
Will do.  Now that it{s passing, I think we can just remove it.